### PR TITLE
Update configmap in load test for preparation to DNS tests

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/perf-tests/pull/1526/files#r511885934

### CODE REVIEW

File: clusterloader2/testing/load/configmap.yaml [hunk 1]
```diff
@@ -6,7 +6,29 @@ metadata:
 immutable: true
 {{end}}
 data:
-  data.yaml: |-
-    a: 1
-    b: 2
-    c: 3
+  all-queries: |
```
Comment (mborsz) [inline clusterloader2/testing/load/configmap.yaml:9]: Please add comment that this is for dns tests + add a comment that other use cases can be added here next to the all-queries file.

File: clusterloader2/testing/load/deployment.yaml [hunk 2]
```diff
@@ -51,6 +51,11 @@ spec:
         - name: configmap
           configMap:
             name: {{.BaseName}}-{{.Index}}
+{{if false}}
```
Comment (mborsz) [inline clusterloader2/testing/load/deployment.yaml:54]: Do we need to mount only selected items? I think it makes sense to mount all and using only /var/configmap/feature-name if needed.

File: clusterloader2/testing/load/configmap.yaml (hunk 1 above)
Comment (wojtek-t) [reply clusterloader2/testing/load/configmap.yaml:12]: done

File: clusterloader2/testing/load/deployment.yaml (hunk 2 above)
Comment (wojtek-t) [reply clusterloader2/testing/load/deployment.yaml]: I wanted to have minimal changes, but yeah - we can go with the eventual solution from the beginning.

**Pull request merged**